src/DataFramesMeta.jl

```diff
@@ -44,6 +44,178 @@ replace_syms!(e::Expr, membernames) =
         mapexpr(x -> replace_syms!(x, membernames), e)
     end
 
+"""
+    @col(kw)
+
+    
+`@col` transforms an expression of the form `z = :x + :y` into it's equivalent in 
+DataFrames's "mini-language". Functions act column-wise. For a row-wise functions, see 
+`@row`
+
+### Details
+
+Parsing follows the same convention as other DataFramesMeta macros, such as `@with`. All 
+terms in the expression that are `Symbols` are treated as columns in the DataFrame, except 
+`Symbol`s wrapped in `^`. To use a variable representing a column name, wrap the variable 
+in `cols`. 
+
+`@col` constructs an anonymous function based off the given expression. It then creates
+a `source => fun => destination` pair that is suitable for the `select`, `transform`, and 
+`combine` functions in DataFrames. 
+
+### Examples
+
+```
+julia> @col z = :x + :y
+[:x, :y] => (##595 => :z)
+```
+
+In the above example, `##595` is an anonymous function equivelent to the following 
+
+```
+(_x, _y) -> _x + _y
+```
+
+```
+julia> df = DataFrame(x = [1, 2], y = [3, 4]);
+
+julia> DataFrames.transform(df, @col z = :x .* :y)
+2×3 DataFrame
+│ Row │ x     │ y     │ z     │
+│     │ Int64 │ Int64 │ Int64 │
+├─────┼───────┼───────┼───────┤
+│ 1   │ 1     │ 3     │ 3     │
+│ 2   │ 2     │ 4     │ 8     │
+
+julia> DataFrames.transform(df, [:x, :y] => ((_x, _y) -> _x .* _y) => :z)
```

Comment:
This one probably isn't needed here, I think everybody can see what the previous one means. :-)